### ISSUE TITLE
question about v prediction

### ISSUE THREAD

**Issue by yiyin-s** [LTH14/JiT#26]
Hi, thank you for the wonderful work!
I am trying to replicate the experiments on e prediction and v prediction from the paper. In my test, the loss of v prediction is smaller than x prediction, which is different from the fig7. in the paper, and the model can't generate anything, just black image.

Following is the loss and sampling code, did I miss something? thank you 

```
    def forward(self, x, labels):
        parameterization = self.parameterization
        assert parameterization in ['x', 'eps', 'v']  
        labels_dropped = self.drop_labels(labels) if self.training else labels

        t = self.sample_t(x.size(0), device=x.device).view(-1, *([1] * (x.ndim - 1)))
       e = torch.randn_like(x) * self.noise_scale

        z = t * x + (1 - t) * e
        dt = (1 - t).clamp_min(self.t_eps)
        v = (x - e) * ((1 - t) / dt)

        if parameterization == 'x':
            target = x
        elif parameterization == 'eps':
            target = e
        elif parameterization == 'v':
            target = v

        output = self.net(z, t.flatten(), labels_dropped)

        # l2 loss
        loss = (target - output) ** 2
        loss = loss.mean(dim=(1, 2, 3)).mean()

        return loss
```

```
    def _forward_sample(self, z, t, labels):
        # conditional
        v_cond = self.net(z, t.flatten(), labels)

        # unconditional
        v_uncond = self.net(z, t.flatten(), torch.full_like(labels, self.num_classes))

        # cfg interval
        low, high = self.cfg_interval
        interval_mask = (t < high) & ((low == 0) | (t > low))
        cfg_scale_interval = torch.where(interval_mask, self.cfg_scale, 1.0)

        return v_uncond + cfg_scale_interval * (v_cond - v_uncond)
```
